README.md

```diff
@@ -64,20 +68,24 @@ more details about the problem in case of failure).
  * **Coverage**
 
     DrNED is capable of reporting how big part of the device model your tests have
-    covered; the tool implements a simple wrapper around this capability, including
-    a status data providing the coverage report in a structured form.
+    covered; the tool implements a simple wrapper around this DrNED capability, 
+    including status data providing the coverage report in a structured form.
 
 ## Debugging issues
 
-Your main tools for debugging issues are logs. First you have the DrNED log that
-you control the filtering of via `/drned-xmnr/log-detail/cli`. Set it to `all`
-when debugging issues. Equally useful is the ncs-python-vm-drned-xmnr.log that you
-control through `/python-vm/logging/vm-levels/drned-xmnr/level/`. To maximize the
-information captured by this log, set to `level-debug`. The log can be found in
-the ncs-python-vm-drned-xmnr.log. It can often be helpful to look into the
-detailed trace of the communication with the (virtual) device. Set the
-`/devices/device/<my-device>/trace` to `pretty` or `raw` to capture the
-communication between the NED and the device.
+Your main tools for debugging issues are logs. 
+
+  * First you have the DrNED log that you control the filtering of via
+    `/drned-xmnr/log-detail/cli`. Set it to `all` when debugging issues. 
```

Comment:
When you are changing this - can you also mention `/drned-xmnr/log-detail/redirect`? You can use that and still keep `log-detail/cli` as `drned-overview` without losing the DrNED output.

Reply:
Good feedback. Updated the text and committed.